docs/en/getting-started/get-started-stack.asciidoc

```diff
@@ -1,9 +1,9 @@
 [[get-started-elastic-stack]]
 == Getting started with the {stack}
 
-Looking for an {stack} ("ELK" tutorial) that shows how to set up the {stack}? In
-this tutorial, you learn how to get up and running quickly. First you install
-the core products:
+Looking for an {stack} ("ELK") guide that shows how to set up the {stack}? 
```

Comment:
The "You learn..." sentence looks a bit odd, so maybe it could be added to this one? For example:
```suggestion
Looking for an {stack} ("ELK") guide that shows how to set up the {stack} and get up and running quickly?
```